### PR TITLE
add bubbles animation style and html structure

### PR DESCRIPTION
**Pull request by adamsobiesak**
I added bubble animation / style css and html .container /, I also added a <link rel = "stylesheet" href = "/ BeerAPI / src / assets / style.css"> for better visualization.

### CODE REVIEW

File: src/assets/animations/bubble.css [hunk 1]
```diff
@@ -0,0 +1,290 @@
+.container {
```
[the remark this answers is not in the document]
Comment (adamsobiesak) [reply src/assets/animations/bubble.css]: I will replace it with .container-bubble for now, but I think it should be in a single container by default.

File: src/index.html [hunk 2]
```diff
@@ -37,7 +38,95 @@ <h2 class="nav__title">I feel like searching for a beer by...</h2>
...
+              </g>
+              <g>
+                  <g transform="translate(497 984)">
+                      <circle cx="15" cy="15" r="15"/>
+                  </g>
+              </g>
+          </g>
+      </svg>
+      </div>
+      <h1>Piwko</h1>
```
[the remark this answers is not in the document]
Comment (adamsobiesak) [reply src/index.html]: its a placeholder to give some height to container, as well as background.

**Comment by adamsobiesak** [comment]
Made some changes on the fly, maybe they help.

File: src/index.html [hunk 3]
```diff
@@ -1,45 +1,95 @@
-<!DOCTYPE html>
```
Comment (ceglarzagata) [inline src/index.html]: nie wiem co tu się stało ale nie masz w pliku html w ogóle calej góry...

File: src/assets/style.css [hunk 4]
```diff
@@ -1,862 +1,2 @@
-@import url("https://fonts.googleapis.com/css?family=Comfortaa:400,700|Open+Sans:400,400i&display=swap");
 
-:root {
```
Comment (ceglarzagata) [inline src/assets/style.css]: usunięty jest też cały plik cssów naszych

**Pull request closed without merging**